patacrep/utils.py

```diff
@@ -75,3 +77,19 @@ def yesno(string):
         string,
         ", ".join(["'{}'".format(string) for string in yes_strings + no_strings]),
         ))
+
+def validate_yaml_schema(data, schema):
+    """
+    Check that the data respects the schema
+    """
```

Comment:
Ailleurs dans le code (sauf erreur de ma part), on utilise plutôt comme convention :

~~~python
def validate_yaml_schema(data, schema):
     """Check that... [short description]

    [long description]
    """
    pass
~~~

Ça serait bien de garder la même convention.